src/network/networkd-sysctl.c

```diff
@@ -22,6 +22,9 @@ static bool common_check(Link *link, int family) {
         if (link->flags & IFF_LOOPBACK)
                 return false;
 
+        if (family != AF_UNSPEC && link->iftype == ARPHRD_CAN)
```

Comment:
can you copy the relevant comment from the commit msg here?

also, shouldn't this rather check `IN_SET(family, AF_INET, AF_INET6) && …`? i mean, you say CAN doesn't support IP, hence check for IP?